packages/workflow/docs/WorflowHeader.stories.tsx

```diff
@@ -0,0 +1,118 @@
+import React from "react"
+import { Meta, StoryFn } from "@storybook/react"
+import { Button } from "@kaizen/button"
+import VisibleIcon from "@kaizen/component-library/icons/visible.icon.svg"
+import { ComponentDocsTemplate } from "../../../storybook/components/DocsContainer"
+import { WorkflowExit, WorkflowHeader, WorkflowHeaderProps } from "../"
+
+const defaultArgs = {
+  workflowName: "Create a self-reflection cycle",
+  stepName: "Settings",
+  status: {
+    content: "Draft",
+    vairant: "statusDraft",
+  },
+  actions: [],
+} satisfies WorkflowHeaderProps
+
+const meta = {
+  tags: ["autodocs"],
+  title: "Components/Workflow Header",
```

Comment:
Might be better to separate it into it's own category,

```suggestion
  title: "Workflow/Header",
```

Reply:
Good catch! That's actually not meant to be there 😆

Reply:
I mean as long as it not confusing to folks when they're looking for it? I know we discussed possibly grouping it under patterns, but I don't know if `patterns/workflow` hierarchy would be more confusing

Reply:
<img width="192" alt="Screenshot 2023-05-18 at 2 34 54 pm" src="https://github.com/cultureamp/kaizen-design-system/assets/36558508/38e8c56f-799f-42ba-a58d-f2639888460b">
What do you think about something like this? @gyfchong - Its similar to the filter button, which is less flat than our standard components, but maybe better for documenting the grouped element. We'd probably also want a progress stepper story that lives inside the footer

Reply:
Or do you think we should scrap the "components" and just do "Workflow Exit"

Reply:
So far my rule has been "If the component can technically be combined with others it should be top level" or rather only nest if it has some kind of hard dependency.

Reply:
Could Stepper be a non-workflow specific component?

Reply:
We could potentially have it as a non-workflow component with some modifications but I don't know if it's something we would want outside of this context. There are limitations from a design perspective where it could be used and I think in time it would be scrapped or revamped - so leaning toward no?

Reply:
Perhaps since we want to guide people to the primary monolith component, we nest the Header, Footer etc. in `Workflow/components/*`

Reply:
I think that's a nice way to describe the relationship / guide folks to the larger wrapper